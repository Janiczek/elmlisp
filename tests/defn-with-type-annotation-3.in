(defn map : (-> (-> a b) a b)
  (fn val)
  "also lying!")
