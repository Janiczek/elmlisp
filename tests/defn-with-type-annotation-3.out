map : (a -> b) -> a -> b
map fn val =
    "also lying!"
